vmaas_sync/vmaas_sync.py

```diff
@@ -121,6 +123,28 @@ def sync_cve_md(webhook_queue, page_size=5000):  # pylint: disable=too-many-bran
                 LOGGER.info('Finished importing CVE metadata (page: %s, page_size: %s, pages: %s)',
                             page, page_size, r_json['pages'])
                 if page >= r_json['pages']:
+                    to_delete = [(cve,) for cve in cves_in_db if cve not in all_rh_cves]
+                    if to_delete:
+                        LOGGER.info('Deleting %s unnecessary CVE metadata', len(to_delete))
+                        execute_values(cur, """select cm.cve from cve_metadata cm
+                                               where not exists(select sv.cve_id from system_vulnerabilities sv
+                                               where cm.id = sv.cve_id)
+                                               and cm.cve in (%s)""",
+                                       to_delete, page_size=len(to_delete))
+                        safety_delete = [cve_name_tuple[0] for cve_name_tuple in cur.fetchall()]
+                        unable_to_delete = [cve for cve in zip(*to_delete).__next__() if cve not in safety_delete]
+                        if safety_delete:
+                            execute_values(cur, """delete from cve_affected_systems_cache
+                                                   where cve_id in (select cve_id from cve_metadata
+                                                   where cve in (%s))""",
+                                           [i for i in zip(*safety_delete)], page_size=len(safety_delete))
+                            execute_values(cur, """delete from cve_metadata where cve in (%s)""",
+                                           [i for i in zip(*safety_delete)], page_size=len(safety_delete))
+                            LOGGER.info('Finished deleting unnecessary CVE metadata')
+                        if unable_to_delete:
+                            LOGGER.warning('Unable to delete %s cves: %s, it present in system_vulnerabilities table',
```

Comment:
`Unable to delete %s cves: %s, it present in system_vulnerabilities table`

the wording is weird, what about changing it to for example:

`Unable to delete %s cves (still referenced from system_vulnerabilities table): %s`